src/feature/dirauth/process_descs.c

```diff
@@ -141,7 +139,6 @@ add_ed25519_to_dir(const ed25519_public_key_t *edkey, authdir_config_t *list,
   if (ed25519_validate_pubkey(edkey) < 0) {
     char ed25519_base64_key[ED25519_BASE64_LEN+1];
     ed25519_public_to_base64(ed25519_base64_key, edkey);
-    log_warn(LD_DIRSERV, "Invalid ed25519 key \"%s\"", ed25519_base64_key);
```

Comment:
Please don't delete this warning: you're hiding failures for every caller of this function.

Instead, do a minimal length check in dirserv_load_fingerprint_file().

Reply:
b6465c0aec08febe5d40dd343838095cb96faf1d